cdap-distributions/src/packer/cdap-sdk-ubuntu12-with-uri.json

```diff
@@ -126,7 +126,7 @@
       "skip_install": true,
       "json": {
         "cdap": {
-          "version": "3.4.0-SNAPSHOT-1",
+          "version": "3.5.0-SNAPSHOT-1",
```

Comment:
It is probably the ``release`` field. But I'll let @dereklwood @wolf31o2 confirm